docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3'
```

Comment:
are container boot orders still a thing? they might have gotten deprecated in spec 3 but it's helpful to order the mongodb container to boot first so that you don't get initial error messages about the application failing to make a connection even though it's just starting up. Not particularly important though.

Reply:
Hey, John here. According to investigation it could become a problem potentially. We will make a commit soon. Meanwhile, we have never had this issue on our own machine before, so if you did have this issue, could you please post the specific error message you mentioned so that we have a clearer view about this? Cheers.

Reply:
it should be fine then. i've just run into problems with it in other projects